### ISSUE TITLE
 The genetic testing data of pet

### ISSUE THREAD

**Issue by yvonne-23MF** [filecoin-project/notary-governance#634]
## Application information 

Link to existing LDN application(s)
- _link to application(s) in
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1121
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1122

  
Are you the data owner?
- [x] Yes
- [ ] No

Is the data private i.e., encrypted or not meant to be retrieved by anyone else on the network?
- [x] Yes
- [ ] No

## Data owner information
- Data owner: Chengdu 23mofang Biotechnology Co., Ltd.
- Data owner's website: https://www.23mofang.com/
- Data owner's social media account(s): Official  Wechat:23魔方   Official Tiktok:@23魔方溯源故事、@23魔方寻宗

### Data owner description 
_Details on the data owner background, source of funding, etc._

Chengdu 23mofang Biotechnology Co., Ltd. was established in March 2015 to provide biogenetic testing and life data analysis services.

According to the "2018 China Genetic Testing Industry Analysis" released by a third-party data agency—Analysys International,23mofang ranked first in the consumer-grade genetic testing field.Here is the news address:https://www.jiemian.com/article/2244613.html

And in In the Forbes list of "Chinese Fifty Most Innovative Companies",23mofang was rated as the "The innovator of Gene testing",check:https://m.huanqiu.com/article/9CaKrnK9GMf。
So far, 23mofang has accumulated 6 rounds of financing from SBCVC,Hanvon technology，Matrix Partners China,CD Capital,etc.23mofang is the largest domestic consumer-based company.

Based on leading genetic research results, 23mofang owns a professional bio-testing center and uses world-leading testing equipment for sequencing, our company provides users with highly readable and usable genetic test reports. 23mofang's self-built genetic testing center has been certified by the National Health and Family Planning Commission and meets all the standards of medical and clinical trial institutions.

For better scientific research,the company also build cooperation with Ministry of Education Key Laboratory of Contemporary Anthropology,the Medical Heredity Research Center of Southwest hospital of AMU,Kunming institute of zoology . cas.

### What is the motivation behind onboarding this data to Filecoin? 

We have applied general verification datacap from notaries before, check [#427](https://github.com/filecoin-project/filecoin-plus-client-onboarding/issues/427)、[#428](https://github.com/filecoin-project/filecoin-plus-client-onboarding/issues/428)、[#602](https://github.com/filecoin-project/filecoin-plus-client-onboarding/issues/602),so we have build trust with Filecoin,and we hope that we can storage our data in Filecoin steady.

### What organizations are involved in this project? Sources of funding, SPs partnering in onboarding, other service providers, etc.?

We currently have an established relationship with MDM, and depending on the subsequent progress of storage and the number of seal, we will continue to investigate and choose a wider range of SPs for cooperation.

## Transparency in KYC
_If you are not the data owner, please email filplus@fil.org with you and the data owner on a thread so at least the Fil+ governance team can act as a lightweight KYC check._

Are you going to offer NDAs to notaries?
- [x] Yes
- [ ] No
  
## Dataset description 
_What is being stored on the network? Please provide as many details as possible on the dataset and its size._
Animal gene test samples, analysis results and gene source data. They are all private data.We provide more than 2P of pet genetic data, and copy to 5 duplicates.

### Why is it useful to the Filecoin network?
_Please share details on how your project may use Filecoin in a novel way, extend or experiment with different use cases, or result in useful development in the ecosystem._
23mofang's  pet genetic testing uses saliva to collect  sample and provides hundreds of test items such as pedigree analysis, genetic risk, genetic traits, genetic variant carriage, nutritional needs, medication response, exercise health, skin management and more. we collect and analyze pet genetic data for diagnosing or ruling out the suspected genetic diseases, and predicting the risk of specific diseases, so we can help users to understand the ancestral origin, physical characteristics and health conditions of pets.

Our pet genetic testing helps humans understand animals better and establish a harmonious ecological environment between humans and animals, while the huge pet test analysis data also greatly promotes the development of animal medical research and animal genetic science research.
  
## Storage needs

Is the data to be stored in a single region, either due to geopolitical constraints or data owner policy?
- [ ] Yes
- [x] No

### Which SPs will you/the data owner be working with to store the data? 
_If all the SPs have not been chosen at the time of issue submission, feel free to edit and update the form as they are chosen. If you need support on finding reputable SPs, please check out filgram.io, filrep.io, plus.fil.org/miners, or Filecoin Slack channels: #fil-plus, #fil-deal-market_
  
| SP ID | SP org | SP region |
| --- | --- | --- |
| f01877862 |MDM | Singapore |
| f01943941 |  | Greater China |
| f0150816   |  | Greater China |
| f01943910   |  | US |
| f01945296   |  | Japan  |

### How were these SPs identified?

We investigate SP from the following two aspects:
--Experience in storing real data 
--Long-term and stable storage operation and maintenance capabilities.

## Notaries that support the project 
 _Notaries perform KYC/KYB to validate the legitimacy of data and organizations. To find supporting notaries please view the list of [round 3 notaries](https://github.com/filecoin-project/notary-governance/tree/main/notaries) and post your LDN application in the Filecoin slack channel #fil-plus-application-review. Feel free to update the form as you gain support over time and more notaries are willing to support your application._

| Notary name | Notary org | Notary region |
| --- | --- | --- |
| _name_ | _org_ | _region_ |

**Comment by Kevin-FF-USA** [comment]
Tagging in @raghavrmadya and @galen-mcandrew @simonkim0515  for a possible re-route of this issue

**Comment by Fenbushi-Filecoin** [comment]
KYB info looks all good from Qcc.

For Storage needs, the data is not to be stored in a single region, then requires SPs to be diversified in different regions.

**Comment by yvonne-23MF** [comment]
> KYB info looks all good from Qcc.
For Storage needs, the data is not to be stored in a single region, then requires SPs to be diversified in different regions.

@Fenbushi-Filecoin   Thank you for your support.
I have updated the sp information in  the proposal.
The SPs we have identified as having been contacted so far are as follows：
SPs in China:
f01943941
F0150816 (SP explained that the node is migrating due to public policies, and we will update new in time)
SP in Singapore :
f01877862
SP in US:
f01943910
SP in Japan :
f01945296
